packaging/s3plugin_lib.py

```diff
@@ -32,3 +32,29 @@ def make_arbitrary_file(f_name, f_size, buffer_size=32*1024*1024):
             out.write(bytearray(buffer))
 
             bytes_written += to_write
+
+def crc64_nvme(data: bytes, initial_crc = 0):
```

Comment:
Consider using more words in the name of the function, e.g. `calculate_crc64_nvme_checksum`

Reply:
done